examples/pyomo_example/pyomo_example.py

```diff
@@ -0,0 +1,66 @@
+from src.FoKL import FoKLRoutines
+import numpy as np
+import pyomo.environ as pyo
+from pyomo.environ import *
+
+
+def main(scenarios):
+    res = int(1e2)
+    t = np.linspace(0, 1, res)
+    window = np.sin(t * np.pi)
+
+    x0 = np.sin(t * np.pi * 3)
+    x1 = np.cos(t * np.pi * 3)
+
+    def b1(x):
+        return -11.2291203558315 + 22.4582407116631 * x
+
+    def b2(x):
+        return 3.93718725725502 - 23.6231235435301 * x + 23.6231235435301 * x ** 2
+
+    def b3(x):
+        return -0.960140559513144 + 11.5216867141577 * x - 28.8042167853943 * x ** 2 + 19.2028111902629 * x ** 3
+
+    y = -window * (b1(x0) + b2(x1) + b2(x0) * b3(x1))
+
+    try:
+        f = FoKLRoutines.load('pyomo_example.fokl')
+    except Exception as e:
+        f = FoKLRoutines.FoKL(kernel=1, draws=2000)
+        f.fit([x0, x1], y, clean=True)
+        f.save('pyomo_example.fokl')
+
+    f.coverage3(draws=scenarios, plot=True, xlabel='t', xaxis=t, ylabel='y')
+
+    m = f.to_pyomo(scenarios)
+
+    # additional constraints to enforce 'physics':
+    m.t = pyo.Var(within=pyo.Reals, bounds=[0, 1])

```

Comment:
Add `, initialize=0.5` to this line

Reply:
This finds the correct solution, but fails if `initialize=0.0` or `initialize=1.0` ... must be finding the local max and 0.5 happens to be close enough in this example

Reply:
resolved because IPOPT is meant to look for *local*, so functioning as expected

will likely add the initialize step inside FoKLRoutines such that inputs x0, x1, ..., xM are initialized at 0.0 by default, and user must choose appropriate solver (i.e., one that looks for *global* max in this example)